ports/psoc6/mpconfigport.mk

```diff
@@ -1,2 +1,2 @@
 MICROPY_VFS_LFS2=1
-
+MICROPY_VFS_FAT=1
```

Comment:
Given that FAT is not really working for the flash (internal or external), and we are going to mount LFS2 in those anyhow, should we just enable this for the boards with SDCard enabled? In the CY8CPROTO-062** boards mpconfigboard.mk.

Reply:
yes moved to 62's board config